index.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Crete+Round&family=Poppins&family=Roboto:wght@700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="styles.css" />
+    <script
+      src="https://kit.fontawesome.com/f1193e9575.js"
+      crossorigin="anonymous"
+    ></script>
+  </head>
+  <body>
+    <header class="container">
+      <nav>
+        <section>
+          <span><a href="#">My logo</a></span>
+          <i class="fa fa-bars"></i>
+        </section>
+      </nav>
+      <article>
+        <h1 class="introduction">Hey There. I'm Amanuel</h1>
+        <h2 class="job-title">I'm a Software Developer</h2>
+        <p class="description">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don't hestiate to contact me.
+        </p>
+        <section class="contact">
+          <a class="logo1" href="https://github.com/amexabee"
+            ><i class="fa fa-github"></i
+          ></a>
+          <a
+            class="logo2"
+            href="https://www.linkedin.com/in/amanuel-galema-56900024a/"
+            ><i class="fa fa-linkedin"></i
+          ></a>
+          <a class="logo3" href="https://angel.co/"
+          ><i class="fa fa-angellist"></i
+        ></a>
+        <a class="logo4" href="https://twitter.com/fountfillwblood"
+          ><i class="fa fa-twitter"></i
+        ></a>
+        <a class="logo5" href="https://twitter.com/fountfillwblood"
+          ><i class="fa fa-medium"></i
+        ></a>
```

Comment:
[Optional]
- [ ] It's better to arrange the social media icons as a list `ul` where each list item `li` has an `a` tag with the icon inside it. This will make your project extremely clean and semantically correct ✨